Gemfile

```diff
@@ -37,6 +37,9 @@ gem 'countries'
 #select country info in the ISO standard
 gem 'country_select'
 
+#parse html
+gem 'nokogiri', '~>1.6.8'
```

Comment:
Add gem versions in `Gemfile` seems a duplication with `Gemfile.lock`. Not a good idea. Remove them until they are absolutly necessary.